app/src/pages/inside/dashboardItemPage/dashboardItemPage.jsx

```diff
@@ -85,9 +135,9 @@ export class DashboardItemPage extends Component {
 
   render() {
     const { formatMessage } = this.props.intl;
+
     return (
-      <PageLayout title={formatMessage(messages.pageTitle)}>
-        <AddDashboardButton />
+      <PageLayout Title={this.getTitle()}>
```

Comment:
Please do not define attributes with a capital letter in the beginning. Fix all following cases.